.gitignore

```diff
@@ -1,7 +1,8 @@
 project/target
 project/project
 target
-
+.idea
 src/main/resources/application.conf
 
 .bsp
+.DS_Store
```

Comment:
These files are not part of the project but of your local setup, they should be part of your own global gitignore, not this one

Reply:
Agreed.